modules/mbedtls/CMakeLists.txt

```diff
@@ -180,6 +180,7 @@ zephyr_interface_library_named(mbedTLS)
       ${ZEPHYR_CURRENT_MODULE_DIR}/library/ssl_tls13_keys.c
       ${ZEPHYR_CURRENT_MODULE_DIR}/library/ssl_tls13_server.c
       ${ZEPHYR_CURRENT_MODULE_DIR}/library/ssl_tls.c
+      zephyr_time.c
```

Comment:
This source file does not look like the others in this list.

It looks more like zephyr_init.c, debug.c, shell.c etc.

Let's move it's CMake code next to one of those three (see above in this file).

Reply:
NB: See https://github.com/zephyrproject-rtos/zephyr/pull/66288#discussion_r1452136887

Reply:
moved it